src/test/java/com/icthh/xm/gate/web/rest/LogsResourceIntTest.java

```diff
@@ -48,7 +48,7 @@ public void setup() {
     public void getAllLogs()throws Exception {
         restLogsMockMvc.perform(get("/management/logs"))
             .andExpect(status().isOk())
-            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE));
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON_VALUE));
```

Comment:
why this media type was changed?

Reply:
Media type MediaType.APPLICATION_JSON_UTF8_VALUE marked as deprecated in sources.